test/unit/deployment-files.test.ts

```diff
@@ -16,7 +16,7 @@ import {
 tap.test('ensure the security properties of the deployment files are unchanged', async (t) => {
   t.same(snykConfig.IMAGE_STORAGE_ROOT, '/var/tmp', 'the snyk-monitor points to the correct mounted path');
 
-  const deploymentFiles = ['./snyk-monitor/templates/deployment.yaml', './snyk-monitor-deployment.yaml'];
+  const deploymentFiles = ['./snyk-monitor-deployment.yaml'];
```

Comment:
Agreed here, it may be best to test it with `helm` itself (with `template` or whichever command validated the files)

Reply:
I'm not sure it worths the time now, mostly because it is tested with the other integration tests